package.json

```diff
@@ -136,7 +136,9 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render linters build:api test build:docs bundler build:demoScripts postinstall types",
+    "build": "npm-run-all build:sass sass:render",
```

Comment:
Should we support a script like `build:all` that will run all 3 of these build scripts?

Reply:
That is what `build:release` is for.

Reply:
During a remote build/release this line is run

https://github.com/AlaskaAirlines/auro-hyperlink/blob/dff0c54009e530d081a482316208f299b27ce8fb/package.json#L166

Then these assets are committed back to the repo

https://github.com/AlaskaAirlines/auro-hyperlink/blob/dff0c54009e530d081a482316208f299b27ce8fb/package.json#L116-L123

Because of this we are not required to run a build locally to commit these resources.